api/v1alpha1/envoygateway_types.go

```diff
@@ -351,6 +354,29 @@ type RateLimitMetricsPrometheusProvider struct {
 	Disable bool `json:"disable,omitempty"`
 }
 
+type RateLimitTracing struct {
+	// SamplingRate controls the rate at which traffic will be
+	// selected for tracing if no prior sampling decision has been made.
+	// Defaults to 100, valid values [0-100]. 100 indicates 100% sampling.
+	// +optional
+	SamplingRate *uint32 `json:"samplingRate,omitempty"`
+
+	// BackendRef defines the target trace collector endpoint configuration
+	BackendRef gwapiv1.BackendObjectReference `json:"backendRef"`
+
+	// AppProtocol defines the protocol of provider port in tracing feature.
+	// Only "http"(default) and "grpc" are allowed in this field
+	// +optional
+	AppProtocol *string `json:"appProtocol"`
```

Comment:
AppProtocol is not needed, since it lives inside backendRef

Reply:
you can assume gRPC for now, and add a follow up to parse backendRef

Reply:
I don't quite understand the following points:
+ BackendRef only contains metadata definitions about the target Service. How do I get the protocol from the Service spec?
+ Similarly, we obtain the FQDN through logical splicing and configure it directly in the ratelimit Pod. Will this go through the gateway api layer?  [comment](https://github.com/envoyproxy/gateway/pull/2974#discussion_r1546693021)

Reply:
@arkodg  Hi, I am back. Can you tell me more?

Reply:
hey @ShyunnY sorry the delay on reviewing this PR, I'd like to bring this issue up in the community meeting next week to again discuss `backendRef` vs explicit `Protocol`, `Host` and `Port` fields for this specific API. Since its tied to the RL service, we may not need `backendRef`